schema/crdb/dbinit.sql

```diff
@@ -3943,6 +3944,22 @@ CREATE TABLE IF NOT EXISTS omicron.public.inv_sled_agent (
     -- similar to `usable_hardware_threads` and friends above.
     cpu_family omicron.public.sled_cpu_family NOT NULL,
 
+    -- The path to the boot disk image file.
+    measurement_manifest_boot_disk_path TEXT NOT NULL,
+    -- The source of the zone manifest on the boot disk: from installinator or
+    -- sled-agent (synthetic). NULL means there is an error reading the zone manifest.
+    measurement_manifest_source omicron.public.inv_zone_manifest_source,
+    -- The mupdate ID that created the zone manifest if this is from installinator. If
+    -- this is NULL, then either the zone manifest is synthetic or there was an
+    -- error reading the zone manifest.
+    measurement_manifest_mupdate_id UUID,
+    -- Message describing the status of the zone manifest on the boot disk. If
+    -- this is NULL, then the zone manifest was successfully read, and the
+    -- inv_zone_manifest_zone table has entries corresponding to the zone
+    -- manifest.
+    measurement_manifest_boot_disk_error TEXT,
```

Comment:
A lot of the comments here seem to be copy/paste of the zone manifest comments.  What's different about these fields?

Reply:
These are tracking the measurement manifest specifically. I missed updating the comments because it does look a lot like the zone manifest.

The measurement manifest exists on the boot disk and the non-boot disk and can have a separate error (e.g. the measurement file doesn't exist). As noted before, it also tracks the same mupdate override because trying to pull that to a higher level was a pretty bit mess.